README.md

```diff
@@ -200,6 +200,51 @@ import { renderRichText } from "@storyblok/js";
 const renderedRichText = renderRichText(blok.richtext);
 ```
 
+You can set a **custom Schema and component resolver globally** at init time by using the `richText` init option:
+
+```js
+import { richTextSchema, storyblokInit } from "@storyblok/js";
+
+const mySchema = deepClone(richTextSchema) // you can make a copy of the default richTextSchema
```

Comment:
deepClone? Should we clarify what it is?

Reply:
Maybe... what I wanted to communicate is that it's better to deeply clone the original schema and make modifications over it, otherwise it might update the original schema and lead to unwanted behaviour. How would you explain it?